tests/helpers/test_selector.py

```diff
@@ -858,22 +858,38 @@ def test_language_selector_schema(schema, valid_selections, invalid_selections)
                     "longitude": 2.0,
                     "radius": 3.0,
                 },
+                {"latitude": 1.0, "longitude": "1.0"},
+                {
+                    "latitude": 1,
+                    "longitude": 2,
+                    "radius": 3,
+                },
             ),
             (
                 None,
                 "abc",
                 {},
                 {"latitude": 1.0},
                 {"longitude": 1.0},
-                {"latitude": 1.0, "longitude": "1.0"},
             ),
         ),
     ),
 )
 def test_location_selector_schema(schema, valid_selections, invalid_selections) -> None:
     """Test location selector."""
 
-    _test_selector("location", schema, valid_selections, invalid_selections)
```

Comment:
Why is this required?

Reply:
Could you be more specific, why is what required? 

If you're asking why I added the converter function, it's because the test fails otherwise when the string input is coerced to float, and `_test_selector` fails.